frontend/src/utils/webapi.js

```diff
@@ -13,6 +16,13 @@ import Node from '../Models/node.js';
 var courseId = "42";
 var mainUrl = "";
 
+// TODO(nate): This is a hardcoded user id. change this dynamically during
+// runtime based on which user is actually viewing
+var userId = 1;
+
+//TODO: this is hardcoded. fix plz
+export var classId = "if44ov1fn5a505";
```

Comment:
Also, could we fetch `courseId` from the list of root nodes exposed by the backend? This would be a much cleaner way to do it. So that would eliminate almost-all hard-coding, which is a major win.